resilience4j-spring-boot-common/src/main/java/io/github/resilience4j/circuitbreaker/autoconfigure/AbstractCircuitBreakerConfigurationOnMissingBean.java

```diff
@@ -56,12 +56,18 @@ public CircuitBreakerRegistry circuitBreakerRegistry(
             circuitBreakerConfiguration.createCircuitBreakerRegistry(circuitBreakerProperties,
                 circuitBreakerRegistryEventConsumer);
 
-        // Register the event consumers
-        circuitBreakerConfiguration
-            .registerEventConsumer(circuitBreakerRegistry, eventConsumerRegistry);
+		// Register the event consumers
+		circuitBreakerConfiguration.registerEventConsumer(circuitBreakerRegistry, eventConsumerRegistry);
+		io.vavr.collection.HashMap<String, String> mergedTags = io.vavr.collection.HashMap.empty();
+		circuitBreakerProperties.getInstances().values().forEach(instanceProperties -> {
+			instanceProperties.getTags().forEach(mergedTags::put);
```

Comment:
`io.vavr.collection.HashMap.put()` returns new `HashMap`.
https://github.com/vavr-io/vavr/blob/b19ce1065a74d2364569f1edf0c9773c59e0ff26/src/main/java/io/vavr/collection/HashMap.java#L781-L784

So `mergedTags` is still empty after `forEach`.
I think it have to be changed like `mergedTags = mergedTags.put(key, value)`.

Reply:
a very good catch man ! let me tackle it then